pants.toml

```diff
@@ -140,6 +140,14 @@ config = "src/python/mypy.ini"
 # with our protobuf stubs.
 args = [ "--show-error-codes", "--namespace-packages" ] #, "--verbose" ]
 
+extra_type_stubs = [
+  "types-click==7.1.8",
+  "types-protobuf==3.19.6",
+  "types-redis==4.1.11",
+  "types-requests==2.27.7",
+  "types-toml==0.10.3",
+]
+
```

Comment:
Interesting... I wonder if we can add our other typestubs to this list and remove them from requirements.txt. They would need to be manually updated, but this might also speed up other operations because we wouldn't be constantly downloading them when we set up Pants, load up a venv for Pulumi in our containers, etc.

The [documentation](https://www.pantsbuild.org/docs/reference-mypy#section-extra-type-stubs) suggests that we could get away with this if we don't need a type library at runtime.

Reply:
We don't have any other other explicitly defined `types-` in requirements.txt, just constraints (meaning other libraries declare a dependency).

Reply:
Does it have to be `types-*` only? Do the `mypy_boto_*` libs not count, or are those something different?

Reply:
Oh. Huh! Interesting point. Let me give it a whirl!